zaraz.go

```diff
@@ -60,18 +68,19 @@ const (
 )
 
 type ZarazTool struct {
-	BlockingTriggers []string        `json:"blockingTriggers"`
-	Enabled          *bool           `json:"enabled"`
-	DefaultFields    map[string]any  `json:"defaultFields"`
-	Name             string          `json:"name"`
-	NeoEvents        []ZarazNeoEvent `json:"neoEvents"`
-	Type             ZarazToolType   `json:"type"`
-	DefaultPurpose   string          `json:"defaultPurpose,omitempty"`
-	Library          string          `json:"library,omitempty"`
-	Component        string          `json:"component,omitempty"`
-	Permissions      []string        `json:"permissions,omitempty"`
-	Settings         map[string]any  `json:"settings,omitempty"`
-	Worker           ZarazWorker     `json:"worker,omitempty"`
+	BlockingTriggers []string               `json:"blockingTriggers"`
+	Enabled          *bool                  `json:"enabled"`
+	DefaultFields    map[string]any         `json:"defaultFields"`
+	Name             string                 `json:"name"`
+	NeoEvents        []ZarazNeoEvent        `json:"neoEvents"`
+	Actions          map[string]ZarazAction `json:"actions"`
+	Type             ZarazToolType          `json:"type"`
+	DefaultPurpose   string                 `json:"defaultPurpose,omitempty"`
+	Library          string                 `json:"library,omitempty"`
+	Component        string                 `json:"component,omitempty"`
+	Permissions      []string               `json:"permissions,omitempty"`
+	Settings         map[string]any         `json:"settings,omitempty"`
```

Comment:
not part of your PR, but I noticed that we should not use `omitempty` for Permissions and Settings

Reply:
thanks for flagging! I'll update here anyway as it's small